src/components/Editor/Editor.js

```diff
@@ -24,28 +34,28 @@ export default function Editor() {
           <option value="roboto-mono">Roboto Mono</option>
           <option value="titillium">Titillium</option>
         </select>
-        <label>Font</label>
+        <label>Font:</label>
       </div>
       <div className="form-control">
```

Comment:
You could take advantage of a neat feature here called event bubbling. If you were to add `onChange={(e) => setAlign(e.target.value)` to this <div> element it would pick up any change event from any of the child elements. The event would "bubble" up to the parent element where it is then captured. This could be useful in a case where you have many more <input> elements and make it so you only have to write it once.